com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -780,9 +821,9 @@ public static void DrawPopup(GUIContent label, SerializedProperty property, stri
         /// <summary>
         /// Draw an EnumPopup handling multiEdition
         /// </summary>
-        /// <param name="property"></param>
-        /// <param name="type"></param>
-        /// <param name="label"></param>
+        /// <param name="label">the label</param>
+        /// <param name="property">The data displayed</param>
+        /// <param name="options">Options of the dropdown</param>
```

Comment:
There is no argument named "options" here

Reply:
true, i ll fix that :)